ROMFS/px4fmu_common/init.d/rc.mc_apps

```diff
@@ -8,7 +8,7 @@
 # INAV, which defaults to 0 now. 
 if param compare INAV_ENABLED 1
 then
-	attitude_estimator_ekf start
+	attitude_estimator_q start
```

Comment:
As you want. IMO it's better to use this estimator than old EKF, just need to move some gyro failover logic before merging. In future inav can be moved to this app, and it will be fast, simple and hackable attitude+position estimator. BTW I think we can run estimation/control loop at higher frequency with this setup, something like 400-500Hz should be ok.